usr/bin/ublue-system-flatpak-manager

```diff
@@ -48,6 +48,7 @@ if grep -qz 'fedora' <<< $(flatpak remotes); then
   /usr/lib/fedora-third-party/fedora-third-party-opt-out
   /usr/bin/fedora-third-party disable
   flatpak remote-delete fedora --force
+  flatpak remote-modify flathub --disable --system
```

Comment:
This should probably be external to this condition?
This only runs if fedora repos are present, but we want it to always run?

Reply:
It runs it once on a clean install, that's all we need right? I copy the old yafti behavior, we deleted it there instead of disabling it on the same condition.